compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolResolver.java

```diff
@@ -1390,7 +1397,7 @@ public void visit(BLangUserDefinedType userDefinedTypeNode) {
                     errored = true;
                 }
 
-                if (tempSymbol.type.tag != TypeTags.TYPEDESC) {
+                if (tempSymbol.type != null && tempSymbol.type.tag != TypeTags.TYPEDESC) {
```

Comment:
Can you share the scenario for which this now becomes null?

Reply:
In that situation, we have already issued the previous error for this code segment (errored variable is true).
 Below example.
```Ballerina
 function Listener (string x, Listener e){
    e.message = x;
}
```

Reply:
Sorry, this is related to the PR because I am enabling an old removed test case.
builtin-symbol-negative.bal